certification/pyxis/pyxis.go

```diff
@@ -126,6 +127,59 @@ func (p *pyxisClient) getImage(ctx context.Context, dockerImageDigest string) (*
 	return &data.Data[0], nil
 }
 
+// FindImagesByDigest uses an unauthenticated call to find_images() graphql function, and will
+// return a slice of CertImages. It accepts a slice of image digests. Additional fields
+// for return should be specified in ContainerImage
```

Comment:
Two things:

1) Should `s/ContainerImage/CertImage/?` in this comment? 

2) Could this be reword to include how we go from `[]digests` to `[]CertImage`? Right now it reads to me as an explanation of the method signature more than what the method accomplishes.

Reply:
CertImages is correct. ContainerImage is the internal name given for the graphql query.

And yes, I'll take a stab at a reword.